### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by MizzyDia**
Translated items, achievements, game, guides, rich presence, store, UI into german and Readme update

### CODE REVIEW

File: achievements/de.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "NEW_ACHIEVEMENT_1_0_NAME":  "Alchemie Praktikant",
+  "NEW_ACHIEVEMENT_1_0_DESC":  "Kreiere dein erstes Element mit Alchemie!",
+  "NEW_ACHIEVEMENT_1_1_NAME":  "Entdecke das Entwicklungsteam",
+  "NEW_ACHIEVEMENT_1_1_DESC":  "N/A",
+  "NEW_ACHIEVEMENT_1_2_NAME":  "Das ist eine Falle",
```
Comment (Elanis) [inline achievements/de.json]: Deepl/google translate tell me that this means "This is a trap", is this a mistake ?
In this context, plant = vegetal

File: game/de.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"element_discovered": "Neues Element kennengelernt: {name}",
+
+	"objective": "Ziel",
+
+	"level": "Level",
+	"challenge-won-msg": "Du hast gewonnen! Du hast es entdeckt!",
```
Comment (Elanis) [inline game/de.json]:
```suggestion
	"challenge-won-msg": "Du hast gewonnen! Du hast es entdeckt",
```

File: items/de.json [hunk 3]
```diff
@@ -0,0 +1,202 @@
+{
+  "42": "42",
+  "air": "Luft",
+  "alchemistry": "Alchemie",
```
Comment (Elanis) [inline items/de.json]:
```suggestion
  "alchemistry": "Alchemistry",
```
Here it's the game itself 😄

File: items/de.json [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+  "binary_system": "Binäres System",
+  "bird": "Vogel",
+  "black_hole": "Schwarzes Loch",
+  "blood": "Blut",
+  "boat": "Boot",
+  "bone": "Knochen",
+  "book": "Buch",
+  "box": "Kiste",
+  "bread": "Brot",
+  "breaded_fish": "Fischbrötchen",
```
Comment (Elanis) [inline items/de.json]:
```suggestion
  "breaded_fish": "Fischstäbchen",
```
I think it's this one instead, I let you confirm

File: items/de.json [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+  "furnace": "Ofen",
+  "galaxy": "Galaxie",
+  "garden": "Garten",
+  "generator": "Generator",
+  "geyser": "Geysir",
+  "glass": "Glas",
+  "glasses": "Brille",
+  "gold": "Gold",
+  "grass": "Grass",
+  "greenhouse": "Glashaus",
```
Comment (Elanis) [inline items/de.json]:
```suggestion
  "greenhouse": "Treibhaus",
```

File: items/de.json [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+  "halo": "Heiligenschein",
```
Comment (Elanis) [inline items/de.json]:
```suggestion
  "halo": "Halo",
```
Here's the Halo from the game

**Review by Elanis — CHANGES_REQUESTED** [review]
Hi @ThisChatban , @MizzyDia 

Thanks a lot for your translation, it'll help me a lot :)

I wrote some reviews on few items, but most of them are okay :)

Thank you for already having signed the CLA.

File: game/de.json (hunk 2 above)
Comment (ThisChatban) [reply game/de.json]: Is it possible to have the discovered item in the middle of the sentence? That would make more sense in the german grammar

Comment (Elanis) [reply game/de.json]: Hm, yes, I'll modify the code to do that. Add a %item% for the item name in the right place inside the sentence, I'll modify other languages when changing the code :)

Comment (ThisChatban) [reply game/de.json]: Awesome thank you :)

Comment (Elanis) [reply game/de.json]: You're welcome, thank you for your translation !

**Review by Elanis — APPROVED** [review]

**Comment by Elanis** [comment]
Everything looks okay for me.
Thanks a lot for your pull request, will be added in the game asap !

**Pull request merged**